core/scripts/go.mod

```diff
@@ -35,9 +35,9 @@ require (
 	github.com/umbracle/fastrlp v0.0.0-20220527094140-59d5dd30e722
 	github.com/urfave/cli v1.22.14
 	google.golang.org/protobuf v1.35.1
-	k8s.io/api v0.31.0
-	k8s.io/apimachinery v0.31.0
-	k8s.io/client-go v0.31.0
+	k8s.io/api v0.31.1
+	k8s.io/apimachinery v0.31.1
+	k8s.io/client-go v0.31.1
```

Comment:
I'm slightly afraid of random k8s upgrades, do we need this? @Tofel

Reply:
Since they are only patch bumps, it would seem that k8s is telling us that we need them.

Reply:
I know, but I did run a soak test in the SDLC using these deps and the test passed